src/luascript.cpp

```diff
@@ -2272,6 +2272,8 @@ void LuaScriptInterface::registerFunctions()
 
 	registerMethod("Creature", "canSee", LuaScriptInterface::luaCreatureCanSee);
 	registerMethod("Creature", "canSeeCreature", LuaScriptInterface::luaCreatureCanSeeCreature);
+	registerMethod("Creature", "canSeeGhostMode", LuaScriptInterface::luaCreatureCanSeeGhostMode);
+	registerMethod("Creature", "canSeeCanInvisibility", LuaScriptInterface::luaCreatureCanSeeInvisibility);
```

Comment:
`canSeeCanInvisibility`?
why not simply `canSeeInvisibility`?

Reply:
because i copy paste and not look 😎